cluster/gce/util.sh

```diff
@@ -2362,7 +2362,8 @@ function kube-up() {
     if [[ "${KUBERNETES_SCALEOUT_PROXY:-false}" == "true" ]]; then
       echo "Scaleout proxy public IP: ${PROXY_RESERVED_IP}:8888"
       echo "Scaleout proxy internal IP: ${PROXY_RESERVED_INTERNAL_IP}:8888"
-      sed -i "s/server: https:\/\/.*/server: http:\/\/${PROXY_RESERVED_IP}:8888/" ${KUBECONFIG}
+      cp -f ${KUBECONFIG} ${LOCAL_KUBECONFIG}
+      sed -i "s/server: https:\/\/.*/server: http:\/\/${PROXY_RESERVED_IP}:8888/" ${LOCAL_KUBECONFIG}
```

Comment:
the local_kubeconfig was updated every time the create_kubemark-Master is called. we probably leave it untouched and if any logic is needed in the upper level function, i.e. in the start Kubemark.sh, they can update as needed.

Reply:
It is only updated when scaleout_proxy is true (first time kube-up is called in workflow that deploys proxy and RP)
start-kubemark does not have access to the variables in invoked bash functions. There is no way to export it back and handle this in caller.